File: js_coding_q_and_a/04.js
// Count the number of vowels in a string.

var vowels = ["a", "e", "i", "o", "u"];

var string = "Hello javascript";
const countVowels = (string, vowels) => {
  console.log(string);
  string = string.toLowerCase();
  var count = 0;
  for (let i in string) {
    if (string[i] == " ") continue;

    if (vowels.includes(string[i])) {
      count++;
    }
  }
  return count;
};

console.log(countVowels(string, vowels));

File: py_coding_q_and_a/04.py
#  Count the number of vowels in a string.

vowels = ["a", "e", "i", "o", "u"]

string = "Hello python"

def count_vowels(text,vowels):
    count=0

    for char in text:
        if char == " " : continue

        if char in vowels: count+=1

    return count


print(count_vowels(string,vowels))